examples/layer-browser/package.json

```diff
@@ -5,7 +5,7 @@
     "build": "webpack app bundle.js --env.local --display-error-details"
   },
   "dependencies": {
-    "deck.gl": "^4.0.0-beta.3",
+    "deck.gl": "^4.0.0-rc1",
```

Comment:
@shaojingli Didn't notice this until now. Correct [semver](www.semver.org) versioning is 4.0.0-rc.1 (note the last dot). We should probably republish the RC (make it a `rc.2`).

Hopefully the wildcards will still work and pick up the latest after we change.